src/workerd/io/worker.h

```diff
@@ -125,8 +125,8 @@ class Worker: public kj::AtomicRefcounted {
     ConsoleMode consoleMode = Worker::ConsoleMode::INSPECTOR_ONLY;
     StructuredLogging structuredLogging = StructuredLogging::NO;
     ProcessStdioPrefixed processStdioPrefixed = ProcessStdioPrefixed::YES;
-    kj::String stdoutPrefix = kj::str("stdout:"_kj);
-    kj::String stderrPrefix = kj::str("stderr:"_kj);
+    kj::ConstString stdoutPrefix = "stdout:"_kjc;
+    kj::ConstString stderrPrefix = "stderr:"_kjc;
```

Comment:
FWIW, I'm not sure LoggingOptions currently ever changes for any consumer, so could potentially just be simplified to a passing around a ref to a single constant struct value per process, rather than making copies of it for every isolate, worker, and isolate lock (!).

This PR will probably help a lot, though, by removing those couple extra allocations per isolate lock when the default kj::StringPtr value is used.

Reply:
Yeah, there's tons and tons of optimizations we can do to reduce memory usage and copying. This PR is focused on span-related optimizations, I wanted it to not get too big.